core/services/keeper/upkeep_executer.go

```diff
@@ -141,11 +142,22 @@ func (ex *UpkeepExecuter) processActiveUpkeeps() {
 
 	ex.logger.Debugw("checking active upkeeps", "blockheight", head.Number)
 
+	registry, err := ex.orm.RegistryByContractAddress(ex.job.KeeperSpec.ContractAddress)
+	if err != nil {
+		ex.logger.With("error", err).Error("unable to load registry")
+		return
+	}
+	turnBinary, err := ex.turnBinary(registry, head, ex.config.KeeperTurnLookBack())
```

Comment:
nit. s/turnBinary/turnBlockHash ?

Reply:
changed to turnBlockHashBinary because it is returning the binary string of the hash